code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -120,7 +121,17 @@
 	adjustBruteLoss(-amount)
 	adjustFireLoss(-amount)
 
+#define XENO_OVERHEALTH_PER_WARDING_LEVEL 25
+#define XENO_OVERHEALTH_DECAY_RATE 5
+#define XENO_OVERHEALTH_GEN_RATE 2.5
 
+#define XENO_MAX_OVERHEALTH ( ( warding_aura * XENO_OVERHEALTH_PER_WARDING_LEVEL ) )
+
+/mob/living/carbon/Xenomorph/proc/handle_overhealth()
```

Comment:
It would be better for the procs handling `overhealth` fluctuation not to let it go under or over the threshold, instead of having to police it every tick in `life()`